pkg/plugins/platform/local.go

```diff
@@ -258,6 +258,15 @@ func (l *LocalApp) Start() error {
 	composePath := l.DockerComposeYAMLPath()
 	l.DockerEnv()
 
+	existing, err := l.FindContainerByType("web")
```

Comment:
You're right, this doesn't fully resolve the issues identified in #136. I do think this is the right approach in terms of the check however. It should be possible for a user to stop site 1 version A, make a copy site 1 version B, and start version B. 

I still don't think we should auto-adjust the site name, but I do think we can change the container name pattern to append a unique hash at the end. With Brad's list rework, I don't think we should be depending on the container name pattern at this point, and should instead be leveraging labels, so I think this could be a reasonable change at this stage.